extensions/objects/templates/list-of-sets-of-translatable-html-content-ids-editor.component.html

```diff
@@ -43,3 +23,34 @@
 <div class="oppia-form-error oppia-drag-and-drop-error">
   {{ errorMessage }}
 </div>
+<style>
+  @media screen and (max-width: 700px) {
+  .txt-two-or-more-items {
+    margin-top: 10px;
+    }
+  }
+
+  .col-sm-1 {
+    margin-right: 10px;
+    width: 1rem;
+  }
+
+  .col-sm-10 {
+    border: 1px solid #000;
+    border-radius: 10px;
+    margin-bottom: 5px;
+    margin-left: 10px;
+  }
+
+  p {
```

Comment:
Prefer defining class selectors instead of tags i.e. create a new class if required.